packages/commonwealth/server/routes/createChain.ts

```diff
@@ -163,6 +167,31 @@ const createChain = async (
     eth_chain_id = +req.body.eth_chain_id;
   }
 
+  // cosmos_chain_id is the canonical identifier for a cosmos chain.
+  if (req.body.base === ChainBase.CosmosSDK && req.body.cosmos_chain_id) {
```

Comment:
So, to be clear, we allow requests to create cosmos chain nodes if they simply omit the chain_id? I think in theory we should mandate chain_ids, unless you can think of a reason not to do so.

Reply:
Makes sense. I can make it required.

The only reason I can think of is for new chains who haven't got around to submitting their info to the Registry yet. We may want to create a starter community before they have all their chain info ready to go for the registry.

@jnaviask

Reply:
makes sense to me.